Sources/Testing/Test+Cancellation.swift

```diff
@@ -208,15 +212,13 @@ extension Test: TestCancellable {
   /// test alone, call ``Test/Case/cancel(_:sourceLocation:)`` instead.
   @_spi(Experimental)
   public static func cancel(_ comment: Comment? = nil, sourceLocation: SourceLocation = #_sourceLocation) throws -> Never {
-    try Self.cancel(
-      comments: Array(comment),
-      sourceContext: SourceContext(backtrace: .current(), sourceLocation: sourceLocation)
-    )
+    let skipInfo = SkipInfo(comment: comment, sourceContext: SourceContext(backtrace: nil, sourceLocation: sourceLocation))
```

Comment:
Why is the `nil` backtrace better than the previous `.current()` in this API context?

Reply:
Consistency with test skipping. Per the comment there:

```
      // We don't need to consider including a backtrace here because it will
      // primarily contain frames in the testing library, not user code.
```